inc/namespace.php

```diff
@@ -304,31 +306,16 @@ function log_remote_request_errors( array $request, ?string $type = null ) {
  * Default ElasticPress functionality is to fall-back to MySQL search when queries fail. We want to instead
  * no-op the query when this happens, as we don't want to put lots of load on to MySQL.
  *
- * @param string $request SQL query string.
+ * @param array $posts
```

Comment:
Missing a parameter comment here.
```suggestion
 * @param ?array $posts List of posts or null if request failed.
```